src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -1172,7 +1187,7 @@ namespace HVACUnitaryBypassVAV {
             }
 
             if (CBVAV(CBVAVNum).DXCoolCoilType_Num > 0) {
-                ControlNodeNum = GetOnlySingleNode(
+                int ControlNodeNum = GetOnlySingleNode(
```

Comment:
I'm getting that this is now an unused variable, was that your intention?

Reply:
My intention was to remove top of function declarations but if this now shows an unused variable I'll check it again and delete if needed.